environment/deployments/science-platform/env/integration.tfvars

```diff
@@ -11,7 +11,8 @@ subnets = [
   {
     "subnet_ip" : "10.130.0.0/23",
     "subnet_name" : "subnet-us-central1-01",
-    "subnet_region" : "us-central1"
+    "subnet_region" : "us-central1",
+    "subnet_private_access" = "true"    
```

Comment:
Interesting that the syntax is "=" in dev and ":" here.  Just checking: "true" is correct as opposed to something like `true`, or it doesn't matter?  I see `false` elsewhere.